packages/keycodes/src/index.js

```diff
@@ -211,9 +211,9 @@ export const modifiers = {
  * @type {WPModifierHandler<WPKeyHandler<string>>} Keyed map of functions to raw
  *                                                 shortcuts.
  */
-export const rawShortcut = mapValues(
-	modifiers,
-	( /** @type {WPModifier} */ modifier ) => {
+export const rawShortcut =
+	/* @__PURE__ */
```

Comment:
Is this something supported by all bundlers?

Reply:
Not sure if it's supported by all, but generally, most would support it, as it's declared in the spec:

https://github.com/javascript-compiler-hints/compiler-notations-spec/blob/main/pure-notation-spec.md

also see compatibility table:

https://github.com/javascript-compiler-hints/compiler-notations-spec/blob/main/pure-notation-compatibility.md

Reply:
The webpack + Terser combo also supports it. Webpack adds the comment to code that is pure (typically re-exports) and then Terser uses the comment to eliminate the dead code.